domain/model/modelmigration/import.go

```diff
@@ -223,23 +223,15 @@ func (i importOperation) Execute(ctx context.Context, model description.Model) e
 	return nil
 }
 
-// Rollback will attempt to rollback the import operation if it was
+// Rollback will attempt to roll back the import operation if it was
 // unsuccessful.
-func (i importOperation) Rollback(ctx context.Context, model description.Model) error {
-	// Attempt to rollback the model database if it was created.
+func (i *importOperation) Rollback(ctx context.Context, model description.Model) error {
+	// Attempt to roll back the model database if it was created.
 	modelName, modelID, err := i.getModelNameAndID(model)
 	if err != nil {
 		return fmt.Errorf("rollback of model during migration %w", errors.NotValid)
 	}
 
-	// If the read only model isn't found, we can simply ignore the error.
```

Comment:
This call is the one that was blocking everything, but it should be covered by the `DeleteModel` call below.